[debug] [2024-06-29T10:37:52.281Z] ----------------------------------------------------------------------
[debug] [2024-06-29T10:37:52.298Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\DUBAI\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2024-06-29T10:37:52.299Z] CLI Version:   13.12.0
[debug] [2024-06-29T10:37:52.300Z] Platform:      win32
[debug] [2024-06-29T10:37:52.300Z] Node Version:  v20.15.0
[debug] [2024-06-29T10:37:52.300Z] Time:          Sat Jun 29 2024 13:37:52 GMT+0300 (Eastern European Summer Time)
[debug] [2024-06-29T10:37:52.301Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-06-29T10:37:52.309Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[debug] [2024-06-29T10:37:52.478Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] !  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2024-06-29T10:37:52.479Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[info] i  To change your data collection preference at any time, run `firebase logout` and log in again. 
[info] 
[info] Visit this URL on this device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=702020775&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2024-06-29T10:37:52.781Z] Unable to open URL: Error: spawn UNKNOWN
    at ChildProcess.spawn (node:internal/child_process:421:11)
    at Object.spawn (node:child_process:761:9)
    at module.exports (C:\Users\DUBAI\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\open\index.js:114:34)
    at open (C:\Users\DUBAI\AppData\Roaming\npm\node_modules\firebase-tools\lib\auth.js:135:5)
    at Server.<anonymous> (C:\Users\DUBAI\AppData\Roaming\npm\node_modules\firebase-tools\lib\auth.js:368:13)
    at Object.onceWrapper (node:events:633:28)
    at Server.emit (node:events:531:35)
    at Server.emit (node:domain:488:12)
    at emitListeningNT (node:net:1932:10)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
